### DISCUSSION TITLE
How do I add RSS feeds to my site?

### DISCUSSION THREAD

**Discussion by jarmos-san** [adityatelange/hugo-PaperMod#217] [Q&A] [answered]
Pretty much what the title says, _"How do I add RSS feeds to my site"_.

Couldn't find any documentation for your theme in context to the question. So, any guidance would be much appreciated.

**Comment by adityatelange** [comment, marked as answer]
Its is already enabled on your site https://jarmos.netlify.app/index.xml

This isn't theme related and you may search hugo docs for addtitional info

**Reply by jarmos-san** [reply]
Got it! Thanks for the heads up, I guess I should've looked into the Hugo docs more thoroughly. But again, thanks for your great work on the theme.
